src/components/EnvironmentOverride/DeploymentTemplateOverride.tsx

```diff
@@ -215,6 +215,11 @@ function DeploymentTemplateOverrideForm({ state, handleOverride, dispatch, initi
                     onClick={handleOverride}
                     type="deployment template"
                 />
+                <div className="form__row">
```

Comment:
Lets say I have selected Chart type as "Rollout" while setting global deployment template. 
Now Can I select "Cron Job" chart ref Id while overriding on environment? 
If Not, I can not see any handling for that.

Reply:
fixed